tech-far-hub/package.json

```diff
@@ -29,6 +29,7 @@
     "gatsby-transformer-sharp": "^5.0.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-use": "^17.4.0",
```

Comment:
My favorite utility lib!

Reply:
I learned about it borrowing code from j40 :D